ddtrace/tracer/option.go

```diff
@@ -147,9 +147,9 @@ func statsTags(c *config) []string {
 	if c.hostname != "" {
 		tags = append(tags, "host:"+c.hostname)
 	}
-	if v, ok := c.globalTags[ext.Environment]; ok {
-		if vv, ok := v.(string); ok {
-			tags = append(tags, "env:"+vv)
```

Comment:
One thing to note is that this code was previously remapping ext.Environment to "env" (which is the default value, but potentially could have been changed?).

I didn't keep this functionality since it seemed unintentional, but can add it back to the new logic if needed.

Reply:
I think it might be better to only have some of the global tags in here. Which ones specifically are of interest to you? 

I would be ok with adding `env`, `version` and `service`, but not sure about all of them.

Reply:
Well actually my concern is that the user shouldn't rely on knowing that only specific tags are sent up with the runtime metrics.

I definitely don't want the library to start adding tags that users aren't expecting to be added, but I feel like the intent of using `WithGlobalTag(..)` is that the tag is attached to all metrics reported by the library.

So I think the solution is to send all the global tags up with the runtime metrics. WDYT?

In my specific case, I need to attach an instance id tag to differentiate multiple instances from each other within the same service type and environment.

Reply:
>  I feel like the intent of using WithGlobalTag(..) is that the tag is attached to all metrics reported by the library

The intent behind `WithGlobalTag` was supposed to be only for APM but part of it can be important application-wise, so I do see your point.

> In my specific case, I need to attach an instance id tag to differentiate multiple instances from each other within the same service type and environment.

It sounds like a legit use case to me.

Reply:
Agreed.

We probably *do* want to add one for version too. `tracer.config.version` doesn't live in `tracer.globalTags`